pkg/skaffold/apiversion/apiversion.go

```diff
@@ -0,0 +1,119 @@
+/*
+Copyright 2018 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package apiversion
+
+import (
+	"fmt"
+	"regexp"
+	"strconv"
+)
+
+type ReleaseTrack int
+
+const (
+	alpha ReleaseTrack = 0
+	beta  ReleaseTrack = 1
+	ga    ReleaseTrack = 2
+)
+
+type Version struct {
+	Major   int
+	Minor   int
+	Release ReleaseTrack
+}
+
+var re = regexp.MustCompile(`^skaffold/v(\d)(?:(alpha|beta)(\d))?$`)
```

Comment:
I think you need to escape the first `/` here to `\/` like `^skaffold\/v(\d)(?:(alpha|beta)(\d))?$`


You already have the capture groups here, so you don't have to manually parse it again.

```golang
		m := r.FindStringSubmatch(s)
		major := m[1]
		prereleaseStr := m[2]
		prereleaseNum := m[3]
```

https://play.golang.org/p/udSiQC7eF_p